GiniVision/Classes/CameraViewController.swift

```diff
@@ -67,6 +82,7 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
     fileprivate var controlsView  = UIView()
     fileprivate var previewView   = CameraPreviewView()
     fileprivate var captureButton = UIButton()
+    fileprivate var documentProviderButton = UIButton()
```

Comment:
Is document provider the button for choosing an image from the camera roll? The name doesn't make it clear.